docs/resources/mlflow_model_serving.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_model_serving Resource
```

Comment:
er, do we want to put this under mlflow? i don't remember what we said for SDKs, which should probably apply here? cc @mengxr

Reply:
My 2 cents - given the SRTI rename, shall we call this a databricks_model_serving resource and the subcategory being "serving"?

Reply:
+1 to Vlad's suggestions.

Reply:
It either should go into Compute category, or maybe create a general "Machine Learning" category and merge MLflow with it

Reply:
Yeah general `Machine Learning` category makes more sense imo, renamed accordingly

Reply:
The discussion here is similar to one for sdk apis. We expect to have many different serving products which are not necessarily Machine Learning related. See the response from Patrick in this comment: https://docs.google.com/document/d/1BwVeBmBgDNCCT_EvdOjoMsmbB0xb16ZDaOhYZXzLezg/edit?disco=AAAAouFslHc

Would suggest to be consistent with decision here and have category be 'endpoints'.  If the aim is to be consistent with the left nav in the webapp, then call this category 'serving'.

Reply:
I think `serving` category makes sense, updated accordingly